launch/hdl_graph_slam.launch

```diff
@@ -16,7 +16,7 @@
   <arg name="robot_odom_frame_id" default="robot_odom" />
 
   <!-- transformation between lidar and base_link -->
-  <node pkg="tf" type="static_transform_publisher" name="lidar2base_publisher" args="0 0 0 0 0 0 base_link velodyne 10" />
+  <!--<node pkg="tf" type="static_transform_publisher" name="lidar2base_publisher" args="0 0 0 0 0 0 base_link velodyne 10" />-->
```

Comment:
why is this commented? is it not required anymore?

Reply:
If playing from bag file, the bag has the tfs. If real time, the collect.launch file has the tfs. So no need here.